src/main/resources/view/MainWindow.fxml

```diff
@@ -32,38 +32,36 @@
             <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
           </Menu>
         </MenuBar>
-        <HBox maxHeight="700" minWidth="500.0" prefHeight="300.0" prefWidth="700.0" styleClass="pane-with-border" VBox.vgrow="ALWAYS">
-          <children>
-            <VBox minWidth="300" prefHeight="200.0" prefWidth="511.0" HBox.hgrow="ALWAYS">
-              <children>
-                <StackPane fx:id="commandBoxPlaceholder" styleClass="pane-with-border" VBox.vgrow="NEVER">
-                  <padding>
-                    <Insets bottom="5" left="10" right="10" top="5" />
-                  </padding>
-                </StackPane>
 
-                <StackPane fx:id="resultDisplayPlaceholder" minHeight="100" prefHeight="200" prefWidth="267.0" styleClass="pane-with-border" VBox.vgrow="ALWAYS">
-                  <padding>
-                    <Insets bottom="5" left="10" right="10" top="5" />
-                  </padding>
-                </StackPane>
+        <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
+          <padding>
+            <Insets top="5" right="10" bottom="5" left="10" />
+          </padding>
+        </StackPane>
 
-                <StackPane fx:id="budgetDisplayPlaceholder" minHeight="50" prefHeight="100" maxHeight="100" styleClass="pane-with-border" VBox.vgrow="NEVER">
-                  <padding>
-                    <Insets bottom="5" left="10" right="10" top="5" />
-                  </padding>
-                </StackPane>
-              </children>
-            </VBox>
+        <StackPane VBox.vgrow="NEVER" fx:id="budgetDisplayPlaceholder" styleClass="pane-with-border"
+                   minHeight="50" prefHeight="50" maxHeight="50">
+          <padding>
+            <Insets top="5" right="10" bottom="5" left="10" />
+          </padding>
+        </StackPane>
+
+        <StackPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="pane-with-border"
```

Comment:
Issue of whether theres enough space so that there's no truncation of the command results.

Reply:
gonna remove the vbox.vgrow then